### PR TITLE
Updating DPP annotation description

### PR DESCRIPTION
**Pull request by sthesse**
... based on experts review.

### CODE REVIEW

File: sap-schemas/v2.0/README.md [hunk 1]
```diff
@@ -598,7 +598,13 @@ Constraints:
 
 - Type: `Boolean`
 - Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
-- Description: Property contains potentially sensitive personal data.
+- Description: Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data that is sensitive may differ for different legal areas or industries.
```
Comment (HeikoTheissen) [inline sap-schemas/v2.0/README.md:601]:
```suggestion
- Description: Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data is sensitive may differ for different legal areas or industries. 
```

File: sap-extensions/extensions.json [hunk 2]
```diff
@@ -45,12 +45,12 @@
...
         },
         "x-sap-dpp-is-potentially-sensitive": {
           "type": "boolean",
           "default": true,
-          "description": "Property contains potentially sensitive personal data"
+          "description": "Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data that is sensitive may differ for different legal areas or industries. Examples of sensitive personal data: 1. Special categories of personal data, such as data revealing racial or ethnic origin, political opinions, religious or philosophical beliefs, trade union membership, genetic data, biometric data, data concerning health or sex life or sexual orientation. 2. Personal data subject to professional secrecy. 3. Personal data relating to criminal or administrative offenses. 4. Personal data concerning insurances and bank or credit card accounts."
```
Comment (HeikoTheissen) [inline sap-extensions/extensions.json:53]:
```suggestion
          "description": "Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data is sensitive may differ for different legal areas or industries. Examples of sensitive personal data: 1. Special categories of personal data, such as data revealing racial or ethnic origin, political opinions, religious or philosophical beliefs, trade union membership, genetic data, biometric data, data concerning health or sex life or sexual orientation. 2. Personal data subject to professional secrecy. 3. Personal data relating to criminal or administrative offenses. 4. Personal data concerning insurances and bank or credit card accounts."
```

File: sap-schemas/v3.0/README.md [hunk 3]
```diff
@@ -567,7 +567,13 @@ Constraints:
 
 - Type: `Boolean`
 - Used at: [Schema Object](https://spec.openapis.org/oas/v3.0.3#schema-object)
-- Description: Property contains potentially sensitive personal data.
+- Description: Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data that is sensitive may differ for different legal areas or industries.
```
Comment (HeikoTheissen) [inline sap-schemas/v3.0/README.md:570]:
```suggestion
- Description: Property contains potentially sensitive personal data. Sensitive personal data is a category of personal data that needs special handling. The determination which personal data is sensitive may differ for different legal areas or industries. 
```

**Review by HeikoTheissen — CHANGES_REQUESTED** [review]

**Review by HeikoTheissen — APPROVED** [review]

File: sap-extensions/extensions.json [hunk 4]
```diff
@@ -45,12 +45,12 @@
         "x-sap-dpp-is-potentially-personal": {
           "type": "boolean",
           "default": true,
-          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension."
+          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension. Personal data describes information which is related to any information relating to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person."
```
Comment (ralfhandl) [inline sap-extensions/extensions.json:48]: > information which is related to any information relating to an identified or identifiable natural person

Sounds a bit indirect, maybe
```suggestion
          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension. Personal data describes information which is related to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person."
```

File: sap-schemas/v2.0/README.md [hunk 5]
```diff
@@ -587,7 +587,7 @@ Constraints:
 
 - Type: `Boolean`
 - Used at: [Schema Object](https://spec.openapis.org/oas/v2.0#schema-object)
-- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension.
+- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension. Personal data describes information which is related to any information relating to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person.
```
Comment (ralfhandl) [inline sap-schemas/v2.0/README.md:590]: "information related to information relating to" - shorten as proposed for the first file

File: sap-schemas/v3.0/README.md [hunk 6]
```diff
@@ -556,7 +556,7 @@ Constraints:
 
 - Type: `Boolean`
 - Used at: [Schema Object](https://spec.openapis.org/oas/v3.0.3#schema-object)
-- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension.
+- Description: Property contains potentially personal data. Properties annotated with `x-sap-dpp-field-semantics` need not be additionally annotated with this extension. Personal data describes information which is related to any information relating to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person.
```
Comment (ralfhandl) [inline sap-schemas/v3.0/README.md:559]: "information related to information relating to" - shorten as proposed for the first file

Comment (sthesse) [reply sap-schemas/v3.0/README.md]: Thanks. Applied the proposal with some adjustments by the experts.

**Review by ralfhandl — DISMISSED** [review]

**Review by HeikoTheissen — APPROVED** [review]

**Review by pavelkornev — APPROVED** [review]

File: sap-extensions/extensions.json [hunk 7]
```diff
@@ -45,12 +45,12 @@
         "x-sap-dpp-is-potentially-personal": {
           "type": "boolean",
           "default": true,
-          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension."
+          "description": "Property contains potentially personal data. Properties annotated with x-sap-dpp-field-semantics need not be additionally annotated with this extension. Personal data describes any information which is related to an identified or identifiable natural person (data subject). An identifiable person is one who can be identified, directly or indirectly, in particular by a reference to an identifier such as a name, an identification number, location data, an online identifier, or to one or more factors specific to the physical, physiological, genetic, mental, economic, cultural, or social identity of that natural person."
```
Comment (pavelkornev) [inline sap-extensions/extensions.json:48]: Please generate SAP schemas ! run `npm run generate` and commit those files to this PR. Thx.

**Review by pavelkornev — CHANGES_REQUESTED** [review]

File: sap-extensions/extensions.json (hunk 7 above)
Comment (ralfhandl) [reply sap-extensions/extensions.json:48]: @pavelkornev How about automating that step with a GitHub action?

**Review by pavelkornev — APPROVED** [review]

File: sap-extensions/extensions.json (hunk 7 above)
Comment (pavelkornev) [reply sap-extensions/extensions.json:48]: Yeah, i think we should generate markdown files in general and have just one place where we define things, not 2.

**Pull request merged**